### PR TITLE
Editor speedup fixes

### PR DESCRIPTION
**Pull request by Learath2**
Fix 1 bug, improve usability of angle selector.

### CODE REVIEW

File: src/game/editor/editor.cpp [hunk 1]
```diff
@@ -3045,30 +3045,22 @@ int CEditor::DoProperties(CUIRect *pToolBox, CProperty *pProps, int *pIDs, int *
...
-			if(!Shift && UI()->MouseButton(0) && (activeItem == &pIDs[i] || activeItem == &pIDs[i]+1 || activeItem == &pIDs[i]+2))
-				Value = (Value / 45) * 45;
-			int NewValue = UiDoValueSelector(&pIDs[i], &Shifter, "", Value, pProps[i].m_Min, Shift ? pProps[i].m_Max : 315, Shift ? 1 : 45, Shift ? 1.0f : 10.0f, "Use left mouse button to drag and change the value. Hold shift to be more precise. Rightclick to edit as text.", false, false, 0);
-			if(NewValue != pProps[i].m_Value)
-			{
-				*pNewVal = NewValue;
-				Change = i;
-			}
+
+			int NewValue = UiDoValueSelector(&pIDs[i], &Shifter, "", Value, pProps[i].m_Min, pProps[i].m_Max, Shift ? 1 : 45, Shift ? 1.0f : 10.0f, "Use left mouse button to drag and change the value. Hold shift to be more precise. Rightclick to edit as text.", false, false, 0);
```
Comment (12pm) [inline src/game/editor/editor.cpp]:
```suggestion
			int NewValue = UiDoValueSelector(&pIDs[i], &Shifter, "", Value, pProps[i].m_Min, pProps[i].m_Max, Step, Shift ? 1.0f : 10.0f, "Use left mouse button to drag and change the value. Hold shift to be more precise. Rightclick to edit as text.", false, false, 0);
```

Comment (Learath2) [reply src/game/editor/editor.cpp]: I wasn't really sure what this did, so I didn't really touch it, do you happen to know?

Comment (12pm) [reply src/game/editor/editor.cpp]: it defines in which steps the value updates when you press +/-

**Comment by Learath2** [comment]
It doesn't the buttons are handled below

File: src/game/editor/editor.cpp (hunk 1 above)
Comment (Learath2) [reply src/game/editor/editor.cpp]: No it's not, it's for when you hold your mouse down and drag on the value box.

**Comment by Learath2** [comment]
The last commit has one caveat. Because now the value is constrained you can't reach 359 while dragging with steps of 45. I don't think it is a problem at all tbh.

File: src/game/editor/editor.cpp (hunk 1 above)
Comment (12pm) [reply src/game/editor/editor.cpp]: well you can still use ``Step`` here to make the line shorter xd

**Pull request merged**